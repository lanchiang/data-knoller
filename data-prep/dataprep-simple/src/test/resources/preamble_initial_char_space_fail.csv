#preamble
#preamble2
#preamble3
43 12
3 9
23 48
23 94
54 93
92 12
2 43
 #postambleWithSpace
 #postambleWithSpace
93 43
2 44
43 1
 #postambleWithSpace
94 32
42 55
83 29
93 2
03 12
#postamble4
 #postambleWithSpace