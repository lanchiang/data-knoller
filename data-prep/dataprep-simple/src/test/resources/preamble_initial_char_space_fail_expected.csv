#preamble
#preamble2
#preamble3
43 12
3 9
23 48
23 94
54 93
92 12
2 43
93 43
2 44
43 1
94 32
42 55
83 29
93 2
03 12
#postamble4